### ISSUE TITLE
Add contribution guidance to the readme

### ISSUE THREAD

**Issue by pfwd** [Podcastindex-org/podcast-namespace#377]
This may seem pretty obvious to most but at first I wasn't I wasn't sure if proposals were being accepted via  GitHub Issues or pull requests.
I think the readme needs a section on how to submit a proposal with a brief explanation on the acceptance/rejection process.

**Comment by daveajones** [comment]
Good call. I may have an even better solution shortly. Going to talk about it on the board meeting tomorrow.

**Comment by pfwd** [comment]
@daveajones Great, I'm glad it's being brought up.  It might be worth having a PR or issue template for proposals so a common submission format can be followed.
